### PR TITLE
new unit tests for webhooks

### PR DESCRIPTION
**Pull request by gormal**
New unit tests for webhooks.

### CODE REVIEW

**Review by Sevitas — CHANGES_REQUESTED** [review]
Just a nitpicks

File: Kontent.Ai.Management.Tests/ManagementClientTests/WebhookTests.cs [hunk 1]
```diff
@@ -1,99 +1,242 @@
 ﻿using FluentAssertions;
+using Kontent.Ai.Management.Extensions;
 using Kontent.Ai.Management.Models.Shared;
 using Kontent.Ai.Management.Models.Webhooks;
 using Kontent.Ai.Management.Tests.Base;
 using System;
 using System.Collections.Generic;
+using System.Net.Http;
+using System.Net;
 using System.Threading.Tasks;
```
[the remark this answers is not in the document]
Comment (gormal) [reply Kontent.Ai.Management.Tests/ManagementClientTests/WebhookTests.cs]: fixed

File: Kontent.Ai.Management.Tests/ManagementClientTests/WebhookTests.cs [hunk 2]
```diff
@@ -1,99 +1,242 @@
 ﻿using FluentAssertions;
+using Kontent.Ai.Management.Extensions;
```
[the remark this answers is not in the document]
Comment (gormal) [reply Kontent.Ai.Management.Tests/ManagementClientTests/WebhookTests.cs]: fixed

File: Kontent.Ai.Management.Tests/ManagementClientTests/WebhookTests.cs [hunk 3]
```diff
@@ -1,99 +1,242 @@
...
 using Kontent.Ai.Management.Models.Shared;
 using Kontent.Ai.Management.Models.Webhooks;
 using Kontent.Ai.Management.Tests.Base;
 using System;
 using System.Collections.Generic;
```
[the remark this answers is not in the document]
Comment (gormal) [reply Kontent.Ai.Management.Tests/ManagementClientTests/WebhookTests.cs]: fixed

File: Kontent.Ai.Management.Tests/ManagementClientTests/WebhookTests.cs [hunk 4]
```diff
@@ -1,99 +1,242 @@
...
+using System.Net.Http;
+using System.Net;
```
[the remark this answers is not in the document]
Comment (gormal) [reply Kontent.Ai.Management.Tests/ManagementClientTests/WebhookTests.cs]: fixed

**Pull request merged**